--- /dev/null
+++ ../src/phpCalendar/Calendar.php
@@ -709,7 +709,7 @@
 
             $today_class = ($running_day->format('Y-m-d') == $today->format('Y-m-d')) ? ' today' : '';
 
-            $calendar .= '<td class="day cal-day cal-day-' . strtolower($running_day->format('l')) . ' ' . $class . $today_class . '" title="' . htmlentities($event_summary) . '">';
+            $calendar .= '<td class="day cal-day cal-day-' . strtolower($running_day->format('l')) . ' ' . $class . $today_class . '" title="' . htmlentities(strip_tags($event_summary)) . '">';
 
             $calendar .= '<div class="cal-day-box">';
 
